queries.sql

```diff
@@ -92,4 +96,41 @@ SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 -- Answer to Question 6
 SELECT species, AVG(escape_attempts) FROM animals 
 WHERE date_of_birth > '1990-01-01' and date_of_birth < '2000-12-31' 
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+-- Day 3
+
+-- Answer to Question 1
+SELECT * FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE full_name = 'Melody Pond';
+
+-- Answer to Question 2
+SELECT * FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
+
+-- Answer to Question 3
+SELECT owners.full_name, animals.name FROM owners
+LEFT JOIN animals ON animals.owner_id = owners.id;
+
+-- Answer to Question 4
+SELECT species.name, COUNT(*) FROM species
+LEFT JOIN animals ON animals.species_id = species.id GROUP BY species.id;
+
+-- Answer to Question 5
+SELECT animals.* FROM animals
+LEFT JOIN owners ON owners.id = animals.owner_id
+LEFT JOIN species ON species.id = animals.species_id
+WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
+
+-- Answer to Question 6
+SELECT animals.* FROM animals
+LEFT JOIN owners ON owners.id = animals.owner_id
+LEFT JOIN species ON species.id = animals.species_id
+WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
```

Comment:
- [ Required ] I think for you to list all the animals owned by Dean Winchester that haven't tried to escape you don't need to join the species table with the animal table. The first join is enough to get you the desired result, I would like to suggest that you refactor this query as shown below.

```SQL
  SELECT animals.* FROM animals
  LEFT JOIN owners ON animals.owner_id = owners.id
  WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
```